src/components/IconButton/IconButton.styles.ts

```
@@ -0,0 +1,14 @@
+import { IconButton, styled } from '@mui/material';
+
+import { ICON_BUTTON_SIZE } from './IconButton.constant';
+import { CustomIconButtonProps } from './IconButton.types';
+
+export const StyledIconButton = styled(IconButton, {
+    shouldForwardProp: (prop) => prop !== 'iconSize',
+})<CustomIconButtonProps>(({ theme: { palette, typography }, iconSize }) => ({
+    color: palette.text.primary,
+    padding: 0,
+    '& svg': {
+        fontSize: typography.pxToRem(ICON_BUTTON_SIZE[iconSize!]),
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)